common.blocks/page/page.js

```diff
@@ -33,6 +33,17 @@ provide(BEMDOM.decl(this.name, {
                     this.unbindFromWin('scroll', this._disableScroll);
                     this.bindToWin('scroll', this._onScroll);
                 }, this);
+
+                //block scrolling while popup open
+
+                Modal.on(this.domElem, {modName: 'visible', modVal: '*'}, function(e, data) {
+                    if (data.modVal) {
```

Comment:
lines from 40 to 45 may be rewritten with just
`this.setMod('hidden', data.modVal);`